### ISSUE TITLE
CYPRESS_SPEC in docker doesn't work

### ISSUE THREAD

**Issue by froens** [cypress-io/cypress#21431]
### Current behavior

Running the following command:
`docker run -v <path-to-project-folder>:/e2e -w /e2e --env CYPRESS_SPEC=cypress/integration/myspec.spec.js cypress/included:3.4.0`

Doesn't only run myspec.spec.js

### Desired behavior

Runs Cypress with only myspec as spec.

Essentially the same behavior as:
`npx cypress run --spec cypress/integration/myspec.spec.js`

### Test code to reproduce

`docker run -v <path-to-project-folder>:/e2e -w /e2e --env CYPRESS_SPEC=cypress/integration/myspec.spec.js cypress/included:3.4.0`

### Cypress Version

3.4.0

**Comment by mjhenkes** [comment]
I'm a bit confused here. I don't think we have ever supported the `CYPRESS_SPEC` env. Are you requesting an enhancement to `cypress-docker-images` to support the ability to select specific spec?

Why wouldn't you just overwrite the entrypoint?
```
docker run -v <path-to-project-folder>:/e2e -w /e2e cypress/included:3.4.0 --entrypoint cypress run --spec cypress/integration/myspec.spec.js
```

**Comment by froens** [comment]
Hi mjhenkes

Thank you for answering. I'm fairly new to cypress. To be honest I don't know what is supported.

The documentation seems to mention this blog post a couple of time: [Run Cypress with a single Docker command](https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command/)
It said that I could change cypress behavior by passing the options as environmental variables (e.g. CYPRESS_VIDEO and CYPRESS_baseUrl). I just assumed that all options were exposed as environmental variables.

I'll try out your work around.

Best regards
Frederik
